### PR TITLE
Refactoring so killing of basic tasks works again

### PR DESCRIPTION
**Pull request by jrx**
- Please review and create a new release DRAX 0.4.0
- Updates the Marathon definition file for DC/OS 1.9
- Adds a Job definition to trigger a destruction on an hourly basis

### CODE REVIEW

**Comment by jrx** [comment]
Hi @joerg84 Can you please take a look :)

**Comment by goofiva** [comment]
@jrx  I built the drax executable, (https://github.com/goofiva/drax/releases) deployed fine but got this when trying to hit the /rampage endpoint. Doesn't look like there's much activity for this PR. Any updates?

`$ http POST localhost:7777/rampage
HTTP/1.1 500 Internal Server Error
Content-Length: 20
Content-Type: text/plain; charset=utf-8
Date: Wed, 17 May 2017 10:21:32 GMT
X-Content-Type-Options: nosniff

Failed to list apps`

File: marathon-drax.json [hunk 1]
```diff
@@ -1,20 +1,38 @@
...
+  "id": "/drax",
   "cmd": "chmod u+x drax && ./drax",
+  "instances": 1,
   "cpus": 0.1,
   "mem": 200,
-  "ports": [
-    0
+  "fetch": [
+    {
+      "uri": "https://github.com/dcos-labs/drax/releases/download/0.4.0/drax"
```
Comment (joerg84) [inline marathon-drax.json:9]: Please make sure this release is available.

Comment (joerg84) [reply marathon-drax.json:9]: Ok. We will create release after merging

**Comment by joerg84** [comment]
@goofiva we just tested the rampage endpoint and it worked fine for us.

**Review by joerg84 — APPROVED** [review]

**Pull request merged**